src/cards/capitalist_card.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'card'
```

Comment:
move all `require_relative` to `require_files.rb` and rename it to `banking.rb` or such same name

Reply:
moved